napari/_qt/qt_about.py

```diff
@@ -0,0 +1,48 @@
+import skimage
+import vispy
+import scipy
+import numpy
+
+import qtpy
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QVBoxLayout, QLabel, QDialog, QFrame
+
+import napari
+
+
+class AboutPage(QWidget):
+    def __init__(self, parent):
+        super(AboutPage, self).__init__(parent)
+
+        self.layout = QVBoxLayout()
+
+        # Description
+        self.layout.addWidget(
+            QLabel("<b>napari</b>: a fast n-dimensional image viewer")
+        )
+
+        # Horizontal Line Break
+        self.hline_break1 = QFrame()
+        self.hline_break1.setFrameShape(QFrame.HLine)
+        self.hline_break1.setFrameShadow(QFrame.Sunken)
+        self.layout.addWidget(self.hline_break1)
+
+        # Versions
+        self.layout.addWidget(QLabel("napari, " + napari.__version__))
+        self.layout.addWidget(QLabel("Qt, " + qtpy.__version__))
+        self.layout.addWidget(QLabel("NumPy, " + numpy.__version__))
+        self.layout.addWidget(QLabel("SciPy, " + scipy.__version__))
+        self.layout.addWidget(QLabel("VisPy, " + vispy.__version__))
+        self.layout.addWidget(QLabel("scikit-image, " + skimage.__version__))
```

Comment:
Can we include some of the results of calling  `vispy.sys_info()` - we've been asking our windows users to send us this info when they are having problems - see here https://github.com/napari/napari/issues/558#issuecomment-537569931
and it might be easier to just ask them to open this dialog box, particularly as we start reaching more users that aren't familiar with the command line

The results from there I want are 
```
Platform: Windows-10-10.0.18362-SP0
Python:   3.7.3 (default, Apr 24 2019, 15:29:51) [MSC v.1915 64 bit (AMD64)]
Backend:  PySide2
GL version:  '4.6.0 - Build 25.20.100.7011'
MAX_TEXTURE_SIZE: 16384
```

I think also the `Qt` right now is reporting the qtpy version not the actual Qt version which could be confusing.
```python
from qtpy import QtCore
QtCore.__version__
```
will give the actual Qt version

Reply:
i am adding `vispy.sys_info()` but line length gets really on some lines

Reply:
can you just pull out the those 5 fields from above?

Reply:
i did :)